model/validation.go

```diff
@@ -1042,66 +1042,95 @@ func ValidateConnectTimeout(timeout *duration.Duration) error {
 }
 
 // ValidateProxyMeshConfig checks that the mesh config is well-formed
-func ValidateProxyMeshConfig(mesh *proxyconfig.ProxyMeshConfig) (errs error) {
+func ValidateProxyMeshConfig(mesh *proxyconfig.MeshConfig) (errs error) {
```

Comment:
nit: s/ValidateProxyMeshConfig/ValidateMeshConfig for consistency with `ValidateProxyConfig()` below and the input to this function is `MeshConfig`.

Reply:
Yes, will fix.

Reply:
Done